pandas/core/frame.py

```diff
@@ -10460,6 +10460,118 @@ def quantile(
         result = self._constructor(res)
         return result
 
+    def quantiles(
+        self,
+        q=0.5,
+        axis: Axis = 0,
+        numeric_only: bool = True,
```

Comment:
[cudf's `DataFrame.quantiles`](https://github.com/rapidsai/cudf/blob/7e112689eb33b04d81b1b70ea7c7ba5df08cdfe0/python/cudf/cudf/core/dataframe.py#L5265) doesn't support a `numeric_only` argument, so the effective default is `numeric_only=False`. Any chance we could modify the default here? Is this meant to align with `quantile` arguments?

Note that I can understand the argument for `numeric_only=True`, but it may add a bit of extra pain in Dask :)

Reply:
Yeah, I did this to align with the default for `quantile` - happy to change to `False` if it make sense to the devs